.github/workflows/float8_multi_gpu_test.yml

```diff
@@ -0,0 +1,51 @@
+name: Run Float8 Tests
+
+on:
+  push:
+    branches:
+      - main
+      - 'gh/**'
+  pull_request:
+    branches:
+      - main
+      - 'gh/**'
+
+concurrency:
+  group: float8_test-${{ github.workflow }}-${{ github.ref == 'refs/heads/main' && github.run_number || github.ref }}
+  cancel-in-progress: true
+
+env:
+  HF_TOKEN: ${{ secrets.HF_TOKEN }}
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: H100
```

Comment:
Note for reviewers: multi-gpu tests will only run on 4xH100, so no coverage of SM89 in this set up. I'm not aware of any multi-gpu hosts for sm89 atm, but can look into it